cmd/controller-manager/app/insecure_serving.go

```diff
@@ -30,6 +30,8 @@ import (
 type InsecureServingInfo struct {
 	// Listener is the secure server network listener.
 	Listener net.Listener
+	// optional server name for log messages
+	Name string
```

Comment:
Do we really need this？ kube-controller and cloud-controller start up as two separate processes.

Reply:
ignore this, used for distinct  metrics and healthz